.travis.yml

```diff
@@ -1,11 +1,10 @@
 language: python
 python:
-- '2.7'
 - '3.4'
 - '3.5'
 - '3.6'
 - '3.7'
-- pypy
+- pypy3
 install:
 - pip install -e .[testing,doc]
 script:
```

Comment:
Comment (relevant to other file):
`mitba/python_compat.py` is for compatibility with Python 2 - The file should be deleted and imports from it should be replaced with python 3 "regular" imports

Reply:
In addition, please add entry in the changelog file